eventsLab.js

```diff
@@ -0,0 +1,46 @@
+document.addEventListener("DOMContentLoaded",()=>{
+    let form = document.querySelector("form")
+    let toDoItem = document.querySelector("#toDoItem")
+    let list=document.querySelector("#listTodo")
+    
+    form.addEventListener("submit",(e)=>{
+        e.preventDefault()
+        
+        
+        if(toDoItem.value){
```

Comment:
you can use `toDoItem.trim()` to remove empty spaces, this is a better way to check since `"    "` would give u a truthy value and it would add an item to the to-do list.